main/src/selenium/performance_test.js

```diff
@@ -11,10 +11,79 @@ const driver = new Builder()
   .build();
 
 const MEASUREMENT_TIME = 500;
-const PATH_TO_OUTPUT_DIR = '../../out/';
+const PATH_TO_OUTPUT_DIR = '../../out/mapbox/';
```

Comment:
Same comment of openlayers/mapbox repetition here, you also use `mapbox.html` below. If you use consistent naming (also for `mapbox.js`, `mapbox.html` etc) you should be able to have the name of library as an argument to the performance test and mention it only once.